dataservice/api/genomic_file/schemas.py

```diff
@@ -77,15 +67,6 @@ class Meta(BaseSchema.Meta, IndexdFileSchema.Meta):
             'api.cavatica_task_genomic_files_list', genomic_file_id='<kf_id>'),
         'biospecimen_genomic_files': ma.URLFor(
             'api.biospecimen_genomic_files_list', genomic_file_id='<kf_id>'),
-        'read_groups': ma.URLFor(
-            'api.read_groups_list', genomic_file_id='<kf_id>')
+        'read_group_genomic_files': ma.URLFor(
```

Comment:
This link on its own is mostly useless to the API user when what they really want is the list of read groups.

Reply:
Ditto ^